notebooks/build/community-distributions.ipynb

```diff
@@ -383,8 +397,7 @@
  "metadata": {
   "colab": {
    "collapsed_sections": [],
-   "include_colab_link": true,
-   "name": "Relating Metaculus community distributions: Infections, Deaths, and IFR",
+   "name": "community-distributions.ipynb",
```

Comment:
Ah, no, thanks. I think this happened because I used the source version of the notebook as my base, and the source notebook has the name stripped. 

We might need to figure out a new way of organizing and diffing our Colabs, this way is pretty error-prone...